web_source/travellers/emma_cr.qmd

```diff
@@ -0,0 +1,31 @@
+---
+title: "EmmaCR"
+subtitle: "City Hopper"
+image: FIXME - https://avatars.githubusercontent.com/u/111275194?v=4
```

Comment:
Looks good Emma. Do you want to remove the 'FIX ME' here before merge?

Reply:
Yes!